src/pocketmine/command/defaults/BanCommand.php

```diff
@@ -49,9 +49,14 @@ public function execute(CommandSender $sender, $currentAlias, array $args){
 			return false;
 		}
 
-		$name = ctype_alnum(str_replace("_", "", array_shift($args)));
+		$name = str_replace("_", "", array_shift($args));
```

Comment:
You shouldn't replace "_" symbol. You can just remove str_replace here.
`$name = array_shift($args);
if (!ctype_alnum(str_replace("_", "", $name))) {
    $sender->sendMessage("Error message");
    return false;
}`